skopt/utils.py

```diff
@@ -122,3 +124,383 @@ def load(filename, **kwargs):
     """
 
     return load_(filename, **kwargs)
+
+
+class pseudo_uniform:
+    """
+    Wrapper around the sobel generator to behave partially like
+    SciPy's random number generators.
+    """
+    def __init__(self, loc, scale):
+        self.loc = loc
+        self.scale = scale
+
+    def rvs(self, size=1, random_state=None):
+        rng = check_random_state(random_state)
+
+        # Generate random seed
+        seed = rng.randint(1000)
+        return self.loc + self.scale * i4_sobol_generate(1, size, seed).ravel()
+
+
+# Copied shamelessly from http://people.sc.fsu.edu/~jburkardt/py_src/sobol/sobol.html
```

Comment:
It is "MIT licensed". I think it is compatible with BSD. (http://softwareengineering.stackexchange.com/q/121998)